packages/data-service/package.json

```
@@ -41,7 +41,7 @@
     "lint": "eslint . && prettier --check .",
     "prepare": "npm run compile",
```

I know that it was already there, but can we switch it to what we are doing in other packages: 
```suggestion
    "bootstrap "npm run compile",
    "prepublishOnly": "npm run compile",    
```